arm-network/2016-09-01/swagger/network.json

```diff
@@ -1663,7 +1663,7 @@
             "description": "The name of the network interface."
           },
           {
-            "name": "parameters",
+            "name": "properties",
```

Comment:
I think "properties" is the right name, according to the MSDN documentation: https://msdn.microsoft.com/en-us/library/azure/mt163668.aspx. Let's get someone from the service involved so they can clear this up (since I'm not sure if MSDN doc is updated for this api-version)